StripeFinancialConnections/StripeFinancialConnections/Source/Native/ManualEntry/ManualEntryDataSource.swift

```diff
@@ -44,7 +47,8 @@ final class ManualEntryDataSourceImplementation: ManualEntryDataSource {
         return apiClient.attachBankAccountToLinkAccountSession(
             clientSecret: clientSecret,
             accountNumber: accountNumber,
-            routingNumber: routingNumber
+            routingNumber: routingNumber,
+            consumerSessionClientSecret: consumerSessionClientSecret
```

Comment:
this is the core change: we pass in `consumerSessionClientSecret` for manual entry so it gets attached to Link account